.github/workflows/GithubActionTests.yml

```diff
@@ -35,7 +41,8 @@ jobs:
 
       - name: Build docker container
         run: |
-          docker build -t quay.io/bioconda/bioconda-utils-build-env-cos7:latest ./
+          sed -i "s#COPY ./bioconda-utils/ ./#COPY ./ ./#" ./bioconda-containers/images/build-env/Dockerfile
```

Comment:
This is a bit hacky!
Another solution would be add a build arg to https://github.com/bioconda/bioconda-containers/blob/94fca9227ce2c7d18f85519e3f2c9f18c8e07fdf/images/build-env/Dockerfile#L35 and pass `./` as a value here.

Reply:
I like the build-arg solution a bit more.

Thanks Martin!

Reply:
https://github.com/bioconda/bioconda-containers/pull/66
Successful build in my fork: https://github.com/martin-g/bioconda-containers/actions/runs/7500340991/job/20418829357